src/sentry_value.c

```diff
@@ -1124,6 +1124,44 @@ sentry_value_new_stacktrace(void **ips, size_t len)
     return stacktrace;
 }
 
+sentry_value_t
+sentry_value_new_transaction_context(const char *name)
+{
+    sentry_value_t transaction_context = sentry_value_new_object();
+
+    sentry_transaction_context_set_name(transaction_context, name);
+
+    return transaction_context;
+}
+
+void
+sentry_transaction_context_set_name(
+    sentry_value_t transaction_context, const char *name)
+{
+    sentry_value_t sv_name = sentry_value_new_string(name);
+    if (sentry__string_eq(name, "") || sentry_value_is_null(sv_name)) {
```

Comment:
should the order around `||` not be inversed?  or is `sentry__string_eq` smart enough?

Reply:
ah yeah, these probably need to be reversed.